components/datatypes/financial-account.schema.json

```diff
@@ -10,14 +10,15 @@
   "title": "Financial Account",
   "type": "object",
   "meta:extensible": true,
-  "meta:industry": ["financial_services"],
   "description": "Financial account details, such as owners, balances, and types.",
   "definitions": {
     "financial-account": {
       "properties": {
         "xdm:currentAccountBalance": {
           "title": "Current Financial Account Balance",
-          "$ref": "https://ns.adobe.com/xdm/datatypes/currency",
+          "items": {
```

Comment:
Is this supposed to be an array ? IF yes, we need to have a type too. Please clarify

Reply:
It was not.  It is fixed now.